medusa/tv/series.py

```diff
@@ -1957,7 +1957,7 @@ def save_to_db(self):
                           'quality': self.quality,
                           'airs': self.airs,
                           'status': self.status,
-                          'flatten_folders': self.flatten_folders,
+                          'flatten_folders': self.season_folders,
```

Comment:
You're saving to the database, value should be inverted.
```py
'flatten_folders': not self.season_folders,
```

Reply:
Great catch, fixed.